recipes/bitserializer/all/conanfile.py

```diff
@@ -45,21 +50,33 @@ def _compilers_minimum_version(self):
             "apple-clang": "12",
         }
 
+    def _patch_sources(self):
+        # Remove 'ryml' subdirectory from #include
+        replace_in_file(self, os.path.join(self.source_folder, "include/bitserializer/rapidyaml_archive.h"), "#include <ryml/", "#include <")
+
```

Comment:
```suggestion

    def package_id(self):
        del self.info.settings
```
Need to preserve only options, as it affects the package content.

Reply:
Unfortunately it fails on my local machine:

> ERROR: 'ConanInfo' object has no attribute 'settings'

Reply:
let's do something different in configure() method then

Reply:
Looks like that it's will not work:

> ERROR: bitserializer/0.50: Error in validate() method, line 83
>         if self.settings.compiler.get_safe("cppstd"):
>         ConanException: 'settings.compiler' doesn't exist
> 'settings' possible configurations are none

Just would like to remind that option "with_csv" produces static library (other is 'header only'), is it good at all to clear settings?

Reply:
I've fixed package_id(), could you take a look?
Now it's generates one package for all setting and options except with_csv.

Reply:
Yes, applying on Package ID is a good choice. Please, only consider the method execution order, I commented package_id() method here, because it's executed at this point, so in case you consider validate() or other late methods, the effect will have no real result, as the package ID is always the same.